hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/StoreWorker.java

```diff
@@ -71,6 +69,7 @@
      * @see #calculateHighestStoreTime
      */
     private long lastHighestStoreTime;
+    private boolean stopped;
```

Comment:
this should be volatile. also something like `stopRequested` would be more precise.